Author: Ralf Treinen <treinen@debian.org>
Originator: Ruben Molina <rmolina@udea.edu.co>
Description: Do not set rpath

--- kwave-0.8.2.orig/CMakeLists.txt	2010-05-17 20:57:55.000000000 +0200
+++ kwave-0.8.2/CMakeLists.txt	2010-05-17 20:58:16.000000000 +0200
@@ -178,11 +178,11 @@
 #############################################################################
 ### RPATH settings                                                        ###
 
-SET(CMAKE_SKIP_BUILD_RPATH TRUE)
-SET(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
-SET(CMAKE_INSTALL_RPATH "${KDE4_LIB_INSTALL_DIR}")
-SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-MESSAGE(STATUS "Using ${CMAKE_INSTALL_RPATH} as install RPATH")
+#SET(CMAKE_SKIP_BUILD_RPATH TRUE)
+#SET(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
+#SET(CMAKE_INSTALL_RPATH "${KDE4_LIB_INSTALL_DIR}")
+#SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+#MESSAGE(STATUS "Using ${CMAKE_INSTALL_RPATH} as install RPATH")
 
 #############################################################################
 ### checks for needed header files                                        ###
